recipes/bumpversion/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "bumpversion" %}
+{% set version = "0.5.3" %}
+{% set hash_type = "sha256" %}
+{% set hash = "6744c873dd7aafc24453d8b6a1a0d6d109faf63cd0cd19cb78fd46e74932c77e" %}
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/b/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  entry_points:
+    - bumpversion = bumpversion:main
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - bumpversion
+
+  commands:
+    - bumpversion --help
+
+about:
+  home: https://github.com/peritus/bumpversion
+  license: MIT
+  summary: 'Version-bump your software with a single command!'
+  license_family: MIT
```

Comment:
Unfortunately the package does not include a license file, that's why I didn't include a `license_file:` entry.